db/schema.rb

```diff
@@ -547,6 +583,64 @@
     t.index ["name"], name: "index_decidim_hashtags_on_name"
   end
 
+  create_table "decidim_ideas_idea_versions", force: :cascade do |t|
+    t.string "item_type", null: false
+    t.integer "item_id", null: false
+    t.string "event", null: false
+    t.string "whodunnit"
```

Comment:
Yeah I'm not really sure about these other changes, like @dsklen. I think maybe we should investigate further before merging this in? If you notice this field `whodunnit`, it does seem like this table name is not correct

Reply:
It seems like some of the plans tables are getting re-created as ideas tables? Not the most careful look and I don't think it's a huge deal but I would like to understand why.

Alternatively to fully understanding, we can manually edit the schema file so only the expected change goes in